lib/convection/model/template/resource/aws_elbv2_target_group.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        ##
+        # AWS::ElasticLoadBalancingV2::TargetGroup
+        ##
+        class ELBV2TargetGroup < Resource
+          include Model::Mixin::Taggable
+
+          type 'AWS::ElasticLoadBalancingV2::TargetGroup', :elbv2_target_group
+          property :health_check_interval_seconds, 'HealthCheckIntervalSeconds'
+          property :health_check_path, 'HealthCheckPath'
+          property :health_check_port, 'HealthCheckPort'
+          property :health_check_protocol, 'HealthCheckProtocol'
+          property :health_check_timeout_seconds, 'HealthCheckTimeoutSeconds'
+          property :healthy_threshold_count, 'HealthyThresholdCount'
+          property :matcher, 'Matcher', :type => :hash
```

Comment:
Is this matcher actually a simple JSON object or should it be a resource property?